packages/client/src/components/Dashboard/TeamMembers.jsx

```diff
@@ -0,0 +1,204 @@
+/* eslint-disable */
+
+import React, { useEffect, useState } from 'react';
+import { apiURL } from '../../apiURL';
+import { AddTeamMemberModal } from '../../containers/LandingPage/AddTeamMemberModal';
```

Comment:
The naming here is a bit misleading perhaps. 
A "modal" is normally an element that pops over the content, usually neutralises the background and closes upon specific interaction or outside click. Somebody working with your code would expect a different behaviour than your code implements. 
Would be good to either rename it, or actually make it a modal 🙏🏽 
MUI can help https://mui.com/material-ui/react-modal/